### PR TITLE
Add support for Borika payment form (Bulgaria)

### PR DESCRIPTION
**Pull request by kaloyan-raev**

### CODE REVIEW

File: src/services/autofill.service.ts [hunk 1]
```diff
@@ -540,6 +541,8 @@ export default class AutofillService implements AutofillServiceInterface {
                 exp = fullMonth + partYear;
             } else if (this.fieldAttrsContain(fillFields.exp, 'mmyyyy')) {
                 exp = fullMonth + fullYear;
+            } else if (this.fieldAttrsContain(fillFields.exp, 'cardexpdate')) {
```
Comment (kspearrin) [inline src/services/autofill.service.ts]: I don't think we can rely a `cardexpdate` field to always require this format.

Comment (kaloyan-raev) [reply src/services/autofill.service.ts]: What's the alternative to fill this field in the proper format?

Comment (harogaston) [reply src/services/autofill.service.ts]: I would go with whichever format is more frequent.

Comment (kaloyan-raev) [reply src/services/autofill.service.ts]: So far a field with label "cardexpdate" is only known to be used by the Borika payment system.

**Comment by eliykat** [comment]
Hi @kaloyan-raev, thanks for your submission! I'm currently reviewing and cleaning up old pull requests. If you're still interested in progressing this PR, please:
* sign the Contributor License Agreement (see the link provided by the CLAassistant bot above)
* reply here and tag me (@) so that I can mark it for further review

As this PR is more than 12 months old, I'll close it for now. However, this is just housekeeping. We encourage our community members to get involved and make contributions, so you can comment here even after it's closed and I'll reopen it if required.

**Pull request closed without merging**